addons/stock/models/stock_move.py

```diff
@@ -197,13 +197,14 @@ def _compute_show_details_visible(self):
         multi_locations_enabled = self.user_has_groups('stock.group_stock_multi_locations')
         consignment_enabled = self.user_has_groups('stock.group_tracking_owner')
```

Comment:
would rather use the '''and move.picking_id.picking_type_id.code != 'incoming'''' here